cmsplugin_offices_ai/models/departments.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -:-
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext_lazy as _
+
+from cmsplugin_offices_ai.models.offices import Office
+
+
+@python_2_unicode_compatible
+class Department(models.Model):
+    name = models.CharField(
+        verbose_name=_('department name'),
+        max_length=128,
+        blank=False,
+    )
+    description = models.TextField(
+        verbose_name=_('description'),
+        max_length=512,
+        blank=True,
+    )
+    office = models.ForeignKey(
+        Office,
+        verbose_name=_('office'),
+        related_name='departments_in_office',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+    )
+
+    # Display options
+    ordering = models.PositiveSmallIntegerField(
```

Comment:
This has typically been called `order_no`. To me `ordering` references the type of ordering..